odk-build.rst

```diff
@@ -0,0 +1,177 @@
+ODK Build
+===========
+
+:dfn:`ODK Build` is a form designer with a drag-and-drop user interface. Build is an HTML5 web application and works best for designing simple forms.
+
+.. tip::
+
+  For more powerful form designers, try :doc:`XLSForm <xlsform>`, `Vellum <https://github.com/dimagi/Vellum>`_, or `Kobo <http://www.kobotoolbox.org/>`_, or `Enketo <https://enketo.org/>`_ or `PurcForms <https://code.google.com/archive/p/purcforms/>`_.
+
+.. _use-build:  
+
+Using ODK Build
+----------------
+
+.. _create-form:
+
+Form building
+~~~~~~~~~~~~~~~
+
+- Go to http://build.opendatakit.org and sign in.
+
+.. image:: /img/odk-build/sign-in.png
+   :alt: Image showing sign in window.
+
+.. note::
+  
+  There is a `downloadable version of ODK Build <https://opendatakit.org/downloads/download-category/build/>`_ available.
+
+- Add a new prompt, by dragging the elements from the bottom of the screen onto the blank canvas. 
+- You can remove a prompt by clicking on :guilabel:`X` sign on the prompt.
+
+.. image:: /img/odk-build/remove-prompt.png
+   :alt: Image showing remove prompt option.
+
+- For each prompt, modify its properties on the right portion of the screen. 
+
+.. image:: /img/odk-build/properties.png
+   :alt: Image showing properties section.
+
+- Prompts can also be rearranged through drag and drop.
+
+.. tip::
+
+  When you start building a form on ODK Build, it is named as *Untitled Form*. To rename it click on :guilabel:`rename` next to the form name and give a name to your form.
+
+.. image:: /img/odk-build/rename.png
+   :alt: Image showing rename option.  
+
+.. _build-menu:  
+
+Build Menus
+~~~~~~~~~~~~~~
```

Comment:
Your first section is functional -- building forms.

But then you switch to talking through the UI.

It might be better to re-organize from a functional perspective: Downloading, Exporting, etc.